### PR TITLE
[#173792786] Spid Level Whitelist

### PR DESCRIPTION
**Pull request by BurnedMarshal**
Upgrande `io-spid-commons` to `5.0.0`.
Add configuration for `SPID_LEVEL_WHITELIST` env var

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -132,13 +134,29 @@ export const CLIENT_ERROR_REDIRECTION_URL =
 export const CLIENT_REDIRECTION_URL =
   process.env.CLIENT_REDIRECTION_URL || "/login";
 
+const SPID_LEVEL_WHITELIST = fromNullable(process.env.SPID_LEVEL_WHITELIST)
+  .map(_ => _.split(","))
+  .foldL(
+    // If SPID_LEVEL_WHITELIST is unset, all the spid levels are enabled
```
Comment (gunzip) [inline src/config.ts:140]: I'd default to `["SpidL1", "SpidL2", "SpidL3"]` if NODE_ENV=dev and `["SpidL2", "SpidL3"]` otherwise

**Review by gunzip — CHANGES_REQUESTED** [review]

File: src/config.ts (hunk 1 above)
Comment (BurnedMarshal) [reply src/config.ts:140]: Whit this change in dev mode you cannot test Spid level whitelisting. Could be a problem?

Comment (BurnedMarshal) [reply src/config.ts:140]: Ok, it's only for the default value. I make the changes.

Comment (BurnedMarshal) [reply src/config.ts]: Fixed in 468f737

**Pull request merged**